### ISSUE TITLE
not finding relevant files

### ISSUE THREAD

**Issue by jpkrishnamoorthy** [yoja-ai/yoja#22]
I think reduction in sources is having a detrimental effect...

"can you summarize my chiropractor visits? "
Used to work.
Now it references the low back pain guide and gives a different answer.

**Comment by jagane** [comment]
Yup, it looks like reduced context broke many things. I will fix.
